andela/mrm_api
# #159075759 paginate resource query

**What does this PR do?**
Allows pagination on the room resources
Allows resources to be returned without pagination

**Description of the task to be completed**
A user should specify page number and item per page

<img width="920" alt="screen shot 2018-08-02 at 19 42 30" src="https://user-images.githubusercontent.com/26790578/43597986-4ea3151e-968c-11e8-883d-5f5572b7b227.png">

Allows all room resources to be returned if pagination's page and per page have not been provided

<img width="917" alt="screen shot 2018-08-02 at 19 45 05" src="https://user-images.githubusercontent.com/26790578/43598116-a05083a6-968c-11e8-89b0-4e586368276c.png">

**How should this be manually tested?**
Create room resources with the required mutation
Provide pagination's page and perPage as shown in the image above (first image)
To test functionality without pagination, just run the query normally (see second image)

**What are related Pivotal Tracker stories?**
Story: Paginate resources query
Story ID: #159075759
